### ISSUE TITLE
Реализация метода x{{ aminoNdcId  }}/s/community/stats/user-active-time

### ISSUE THREAD

**Issue by aspectDevel** [gunboxer/AminoC#1]
Привет, ты случаем не разобрался с этим методом?
x{ndc сообщества}/s/community/stats/user-active-time

**Comment by gunboxer** [comment]
Нет.. Я пока мобильное апи забросил заинтересовался апи официального вэбприложения, сделал бота на его базе, пока что кастыльного.

В апи вэбапликухи есть похожий метод на тот, что тебе нужен:

`curl 'https://aminoapps.com/api/community/stats/web-user-active-time' -H 'cookie: ***' -H 'origin: https://aminoapps.com' -H 'accept-encoding: gzip, deflate, br' -H 'accept-language: en,en-US;q=0.9,ru;q=0.8,de;q=0.7' -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36' -H 'content-type: application/json' -H 'accept: */*' -H 'referer: https://aminoapps.com/c/***' -H 'authority: aminoapps.com' -H 'x-requested-with: xmlhttprequest' --data-binary '{"ndcId":"***"}' --compressed`

Ответ:

`{"code": 200, "result": {"api:duration": "0.002s", "api:message": "OK", "api:statuscode": 0, "api:timestamp": "2018-12-11T10:25:55Z"}}`
